cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -258,12 +260,12 @@ handleSimpleNode p trace nodeTracers npm onKernel = do
   createTracers
     :: NodeProtocolMode
     -> Trace IO Text
-    -> Tracer IO GHC.Base.String
+    -> Tracer IO String
     -> Consensus.TopLevelConfig blk
     -> IO ()
   createTracers npm' tr tracer cfg = do
      case npm' of
-       RealProtocolMode (NodeCLI _ rNodeAddr _ runDBValidation) -> do
+       RealProtocolMode NodeCLI{nodeAddr, validateDB} -> do
```

Comment:
So `NamedFieldPuns` are good to go, unlike what was being categorically told by people.

And yes, personally, they make complete sense here.

Reply:
Yes. `NamedFieldPuns` are great. `RecordWildcards` is what makes some people grumpy.